ci/scripts/run-test-network-basic.sh

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 FABRIC_VERSION=${FABRIC_VERSION:-2.2}
 CHAINCODE_LANGUAGE=${CHAINCODE_LANGUAGE:-go}
 CHAINCODE_NAME=${CHAINCODE_NAME:-basic}
+CHAINCODE_PATH=${CHAINCODE_PATH:-../asset-transfer-basic/chaincode-go/}
```

Comment:
The tests are failing because you need to pass the paths of the other chaincode languages, javascript, java, typescript, to this script.

Reply:
Fixed